app/postgis.py

```diff
@@ -0,0 +1,218 @@
+import binascii
```

Comment:
What about... (see, I'm still stuck) http://postgis.net/docs/RT_ST_AsBinary.html in which case it really is just raw bytes.

> By default, WKB output contains the external file path for out-db bands. If the client does not have access to the raster file underlying an out-db band, set `outasin` to TRUE.

I get that it's convenient for a client to get the filename and read the raster itself, but you're not doing that here, are you?